pkg/cli/build-minirootfs.go

```diff
@@ -35,16 +38,17 @@ func BuildMinirootFS() *cobra.Command {
 		Example: `  apko build-minirootfs <config.yaml> <output.tar.gz>`,
 		Args:    cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return BuildMinirootFSCmd(cmd.Context(), args[0], args[1], useProot)
+			return BuildMinirootFSCmd(cmd.Context(), args[0], args[1], useProot, buildDate)
 		},
 	}
 
 	cmd.Flags().BoolVar(&useProot, "use-proot", false, "use proot to simulate privileged operations")
+	cmd.Flags().StringVar(&buildDate, "build-date", "now", "date used for the timestamps of the files inside the image")
```

Comment:
`build-date` defaults to `now`. This means images are not reproducible by default. Is this ok ?

Reply:
Yes, that seems like the best option.  If you want a reproducible image, then ask for a reproducible image.

Reply:
What's the use case for preferring a non-reproducible image?

I think if we can make it reproducible by default there's a lot of nice wins you can get for free. For one, all your pishes get a lot faster because there's nothing new to upload most of the time.

Reply:
People freak out about the date being 1970.  I don't really *care* per se about the default, as much as I do about having an answer for the people who freak out about the date being 1970.

Reply:
Yeah, I think that's just a documentation issue. For example: https://github.com/google/ko/#why-are-my-images-all-created-in-1970

I don't want to make the experience subtly worse for the vast majority of users who don't care and could benefit from reproducible images, just to appease the relatively small (but vocal) minority who notice and for some reason care. For them, there's an escape hatch in `SOURCE_DATE_EPOCH` and/or the flag.

For more precedent, buildpacks also defaults to reproducible timestamps (they picked 1980 but whatever), which they [document](https://buildpacks.io/docs/features/reproducibility/#consequences-and-caveats), and they're only just now [working on making it overrideable](https://github.com/buildpacks/spec/pull/292).

Reply:
okay, let's make the default `0` and then document it

Reply:
FWIW if having 1970 (or any specific old date) is an issue, you could pick the date of the first commit to the apko repository (or any specific date that holds value for the project) as a default value I think.

Reply:
Nah, lets focus on `0`, since that's the standard one.  We can address this as an FAQ item.

Reply:
Should be fixed, thanks everyone!